src/CalcManager/Ratpack/CalcErr.h

```diff
@@ -34,49 +36,51 @@
 // This format is based loosely on an OLE HRESULT and is compatible with the
 // SUCCEEDED and FAILED macros as well as the HRESULT_CODE macro
 
+typedef int32_t ResultCode;
```

Comment:
If you're gonna dump HRESULTs, you might as well keep the annotations by using something like [`_Return_type_success_(return >= 0)`](https://github.com/Microsoft/DirectXShaderCompiler/blob/master/include/dxc/Support/Global.h#L22).

Reply:
This would, however, re-introduce non-standard extensions and thus break platform-independence.